packages/tabs/components/Tabs.tsx

```diff
@@ -14,52 +23,61 @@ injectGlobal`
 }
 
 .react-tabs__tab-list {
-  border-bottom: 1px solid #aaa;
-  margin: 0 0 10px;
-  padding: 0;
+  border-bottom: 1px solid ${themeBorder};
+  ${margin("bottom", "s")}
+  ${padding("vert", "none")}
+  ${padding("horiz", "l")}
 }
 
 .react-tabs__tab {
   display: inline-block;
-  border: 1px solid transparent;
-  border-bottom: none;
-  bottom: -1px;
   position: relative;
   list-style: none;
-  padding: 6px 12px;
+  ${padding("vert", "s")}
+  ${margin("right", "s")}
   cursor: pointer;
+  font-weight: ${fontWeightMedium};
+  color: ${themeTextColorPrimary};
+  &:after{
+    content: "";
```

Comment:
why specify everything here and then hide it with `display: none;`?

i'd vow to either 

```
.base {
  all: the;
  sty: les;
  display: none;
}

.base--selected {
  display: block;
}
```

or 
```
// skip .base-styling

.base--selected {
  all: the;
  sty: les;
  display: block;
}
```

Reply:
I'll adjust.